lampo-vls/src/lib.rs

```diff
@@ -1,20 +1,19 @@
-use lampo_common::bitcoin::Network;
-use lampo_common::conf::LampoConf;
-use lampo_common::keys::KeysManagerFactory;
-use protocol_handler::InProcessProtocolHandler;
-use signer::VLSSignerPort;
+pub mod protocol_handler;
+pub mod signer;
+
 use std::str::FromStr;
 use std::sync::Arc;
-use triggered::{trigger, Listener};
-use url::Url;
+
 use vls_proxy::portfront::SignerPortFront;
 use vls_proxy::vls_frontend::frontend::SourceFactory;
 use vls_proxy::vls_frontend::Frontend;
 use vls_proxy::vls_protocol_client::KeysManagerClient;
+use triggered::{trigger, Listener};
+use url::Url;
 
-pub mod protocol_handler;
-pub mod signer;
-
+use lampo_common::bitcoin::Network;
+use lampo_common::conf::LampoConf;
+use lampo_common::keys::KeysManagerFactory;
```

Comment:
Mh not sure that this is the right position where to add a key manager factory, you are bringing inside the lampo common vls, but vls should be optional